optimum/exporters/onnx/utils.py

```diff
@@ -190,3 +212,50 @@ def _get_submodels_and_onnx_configs(
         model_kwargs,
         exporter="onnx",
     )
+
+
+def get_stable_diffusion_models_for_export(
+    pipeline: "StableDiffusionPipeline",
+    int_dtype: str = "int64",
+    float_dtype: str = "fp32",
+) -> Dict[str, Tuple[Union["PreTrainedModel", "ModelMixin"], "ExportConfig"]]:
+    logger.warn(
+        "usage `optimum.exporters.onnx.utils.get_stable_diffusion_models_for_export` deprecated, please use `optimum.exporters.utils.get_stable_diffusion_models_for_export` instead"
+    )
+    return _get_stable_diffusion_models_for_export(pipeline, int_dtype, float_dtype, exporter="onnx")
+
+
+def get_sam_models_for_export(model: Union["PreTrainedModel", "TFPreTrainedModel"], config: "ExportConfig"):
+    logger.warn(
```

Comment:
`warn` is deprecated, can you change to `warning`?

Reply:
updated